third_party/move/move-vm/runtime/src/loader/function.rs

```diff
@@ -37,16 +37,16 @@ pub(crate) struct Function {
     pub(crate) file_format_version: u32,
     pub(crate) index: FunctionDefinitionIndex,
     pub(crate) code: Vec<Bytecode>,
-    pub(crate) type_parameters: Vec<AbilitySet>,
+    pub(crate) ty_arg_abilities: Vec<AbilitySet>,
     // TODO: Make `native` and `def_is_native` become an enum.
     pub(crate) native: Option<NativeFunction>,
     pub(crate) def_is_native: bool,
     pub(crate) def_is_friend_or_private: bool,
     pub(crate) scope: Scope,
     pub(crate) name: Identifier,
-    pub(crate) return_types: Vec<Type>,
-    pub(crate) local_types: Vec<Type>,
-    pub(crate) parameter_types: Vec<Type>,
+    pub(crate) return_tys: Vec<Type>,
+    pub(crate) local_tys: Vec<Type>,
+    pub(crate) arg_tys: Vec<Type>,
```

Comment:
The `types -> ty` changes look good and I would be open to making it a convention.

The `parameter -> arg` change though, might need to be revisited. I could be wrong, but my impression is that it is a commonly accepted convention within the PL space that
- "Parameters" are used to denote value bindings in function signatures -- in other words, a prototype of the value to be accepted
  - For example, `x` in `fun foo(x: u64)` is a parameter
- "Arguments" are used to denote actual values that get passed to a function
  - For example, `3` in `foo(3)` is an argument

I don't recall any authoritative definitions of these terms, but if you google "function parameters vs arguments" you'll see many people defining them this way.

Reply:
We actually a discussion on this back in the old days, and that resulted in us changing some of the stuff to `type formals/type actuals`. While I personally like those names for their preciseness, they are probably a bit too formal for anyone except maybe the most fervent PL people. We might want to consider changing those to `params/tys` instead.

Reply:
There is one misnaming I know though
```
pub struct StructTag {
    pub address: AccountAddress,
    pub module: Identifier,
    pub name: Identifier,
    // alias for compatibility with old json serialized data.
    #[serde(rename = "type_args", alias = "type_params")]
    pub type_params: Vec<TypeTag>,
}
```
Here the `TypeTag`s are actual arguments to the type constructor, so they should be called arguments instead of parameters.

Reply:
Yes, we have `params` and `args` used interchangeably, I would love us to just use one convention, no strong opinion which one. Let me use `params` then where possible 👍

Reply:
I think being less informal with `params/tys` is better if we want to attract external contributors 😄

Reply:
> we have params and args used interchangeably

Only in some places, and those are definitely cases of misnaming. The rest of the code follows either the "parameter/argument" or the "formal/actual" conventions.